actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -155,7 +155,9 @@ def use_cookies_with_metadata
   #   restrict to the domain level. If you use a schema like www.example.com
   #   and want to share session with user.example.com set <tt>:domain</tt>
   #   to <tt>:all</tt>. Make sure to specify the <tt>:domain</tt> option with
-  #   <tt>:all</tt> or <tt>Array</tt> again when deleting cookies.
+  #   <tt>:all</tt> or <tt>Array</tt> again when deleting cookies. To support
+  #   an explicit list of domains, provide an array, and the first domain 
+  #   matching <tt>request.host</tt> will be used.
```

Comment:
This new sentence seems like it should come before the caveat about deleting cookies.

Reply:
That's a good idea. I had a tough time figuring out where it should live.